.github/workflows/deploy.yml

```diff
@@ -60,3 +60,55 @@ jobs:
           prerelease: ${{ contains(needs.prepare-to-publish.outputs.gem-version, '-') }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  documentation:
+    needs: prepare-to-publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+
+      - name: Install required gem
+        run: gem install yard --no-document
+
+      - name: Build YARD documentation
+        run: yard doc
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs
+
+  documentation:
+    needs: prepare-to-publish
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+
+      - name: Install required gems
+        run: |
+          gem install yard --no-document
+          gem install redcarpet --no-document
+
+      - name: Build YARD documentation
+        run: yard doc
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs
```

Comment:
Why is this in here twice? The only difference I see is the `redcarpet` gem being installed.

Reply:
Whoops! I moved that in from another file, must have pasted it twice